framework/src/interfaces/Coupleable.C

```diff
@@ -40,7 +40,7 @@ Coupleable::Coupleable(const MooseObject * moose_object, bool nodal, bool is_fv)
     _coupleable_neighbor(_c_parameters.have_parameter<bool>("_neighbor")
                              ? _c_parameters.get<bool>("_neighbor")
                              : false),
-    _coupleable_max_qps(_c_fe_problem.getMaxQps()),
+    _coupleable_max_qps(Moose::constMaxQpsPerElem),
```

Comment:
I believe this should be
```
std::max(Moose::constMaxQpsPerElem, _c_fe_problem.getMaxQps())
```
so that it'll still work if the default order is higher. Same in MaterialPropertyInterface.

Reply:
Yeah - that's a good, simple idea.  But then how do we handle the error message?  We don't want to error in cases where the initial max qps is higher than constMaxQpsPerElem, but we do want to error if it is later bumped to be higher than constMaxQpsPerElem.  I'll come up with something.